slirp.h

```diff
@@ -0,0 +1,6 @@
+#ifndef SLIRP_H
+# define SLIRP_H
```

Comment:
nit: indent

Reply:
I did that on purpose as the define is a level below the #ifndef, but if you don't like to indent the C preprocessor macros, I can drop the extra space